kmk/firmware.py

```diff
@@ -22,9 +22,19 @@
 import kmk.types  # isort:skip
 import kmk.util  # isort:skip
 
-# Now handlers that will be used in keys later
+import busio
+import gc
+
 import kmk.handlers.layers
 import kmk.handlers.stock
+# Now handlers that will be used in keys later
```

Comment:
This is generally bad. The import order that was in place was there for a reason - why did _all of the things_ move in this PR?

Reply:
I ran a make fix-isort. Other than that, can't tell you.

Reply:
Moved them back, added tags to not move with isort. Should stay resolved.